recipes/tuplet/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+import os
+from conan import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class TupletConan(ConanFile):
+    name = "tuplet"
+    license = "BSL-1.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/codeinred/tuplet"
+    description = "A fast, simple tuple implementation that implements tuple as an aggregate"
+    topics = ("tuple", "trivially-copyable", "modern-c++")
```

Comment:
```suggestion
    topics = ("tuple", "trivially-copyable", "modern-cpp")
```
Avoid spaces and `+` . Search engines are trick and can understand it as `modern-c` only